src/features/game/GameProvider.tsx

```diff
@@ -10,6 +10,12 @@ import { cacheShortcuts, getShortcuts } from "features/hud/lib/shortcuts";
 
 import { startGame, MachineInterpreter } from "./lib/gameMachine";
 import { InventoryItemName } from "./types/game";
+import { inspect } from "@xstate/inspect";
+
+inspect({
```

Comment:
Should we put some dev flags around this or is this something we want to run in production?

Reply:
This was something I was testing with Spencer, but didn't quite work..
I'll remove it.